app/Http/Controllers/Api/ApiV1Controller.php

```diff
@@ -4494,4 +4494,95 @@ public function statusUnpin(Request $request, $id)
 
         return $this->json($status);
     }
+
+    /**
+     * GET /api/v1/notifications
+     *
+     * @return array
+     */
+    public function accountNotificationsMarkAsRead(Request $request)
+    {
+        /* abort_if(! $request->user(), 403);
```

Comment:
Why is this commented out?

Reply:
This was used to make testing easier, it wasn't supposed to be in PR, I'm going to remove the comment